### ISSUE TITLE
LaTeX builder adds unnecessary split environment

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1299]
I have the following ReST code:

```
#!rest

.. math:: \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}

Test paragraph.

.. math::

   \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}

Test paragraph.
```

This generates the following LaTeX source when I use `make latex`:

```
#!latex

\begin{gather}
\begin{split}\frac{-b \pm \sqrt{b^2 - 4ac}}{2a}\end{split}\notag\\\begin{split}\end{split}\notag
\end{gather}
Test paragraph.
\begin{gather}
\begin{split}\frac{-b \pm \sqrt{b^2 - 4ac}}{2a}\end{split}\notag
\end{gather}
Test paragraph.
```

Notice the empty `\begin{split}\end{split}` pair in the first equation block. This adds an extra blank space after the equation in the PDF, which looks rather ugly in some circumstances. I'd expect both equation blocks to produce identical results, without the empty split environment. If this is intended behavior, it's not documented.

Tested in 1.2b3 and also present in 1.1.3.

Thanks for the otherwise excellent project. I'm using Sphinx to produce a non-fiction book and finding it very useful.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1299
- Originally reported by: Alex Reinhart
- Originally created at: 2013-10-29T02:47:23.183

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-12 20:47:37+00:00_

Thanks for the report!

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-12 20:47:46+00:00_

Closes #1299: Make behavior of the :rst:dir:`math` directive more consistent and avoid producing empty environments in LaTeX output.

→ <<cset f314c4016090c562266333e9804f84cdc6bea5dd>>
